app/Exceptions/TranslatableException.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Exceptions;
+
+use Exception;
+use Throwable;
+
+class TranslatableException extends Exception
+{
+    /**
+     * error .
```

Comment:
I mean... what else is there to be said, right?

```suggestion
     * Error description.
```

Reply:
Updated